### ISSUE TITLE
Project Structure Name Changes

### ISSUE THREAD

**Issue by milesave** [jessehartloff/Comments#1]
I'm not exactly sure why, but MVP before a Beta release doesn't make sense to me personally. I believe the phases could be formatted like this:

1. Team Formation: Due week 2
2. Alpha Release: Due week 6 - this would combine prototype and MVP into a single sprint, where a team shouldn't sit idle during the prototype phase.
3. Beta Release: Due week 8
4. Collaboration: Due week 10
5. Official Release: Due week 12
6. Presentation: Due week 15

**Comment by jessehartloff** [comment]
Agreed. As discussed in class, there was never any communication from me saying that teams members have to sit idle at any time. The idea of branching is that you can work in parallel.
